android/sdk/build.gradle

```diff
@@ -79,9 +87,14 @@ if (!project.hasProperty('ignoreVersionCheck') || !project.getProperty('ignoreVe
 
 android {
     compileSdkVersion project.compileSdkVersion
-    buildToolsVersion project.buildToolsVersion
+//    buildToolsVersion project.buildToolsVersion
     resourcePrefix "weex"
     useLibrary 'org.apache.http.legacy'
+    packagingOptions {
```

Comment:
In order to make Android App with or without `libc++_shared.so` run as expected, you need to do the following things:
1. delete `libc++_shared.so` in `libs` directory
1. Add a command flag `./gradlew -PIgnoreLibC=true` to enable `packagingOptions`. The `packagingOptions` should be disabled as default.